### DISCUSSION TITLE
Improve support for Win10 Cli multi-file selection

### DISCUSSION THREAD

**Discussion by jdonnerstag** [vlang/v#12705] [Ideas]
I'm using Win10 for my stuff and occassionally I want to run all test files in a specific folder, but without any subdirectory. On linux I can do `v.exe test modules\rosie\*_test.v` because linux *shell* will replace the "file" with all the file names matching. Win CLI (cmd) is not doing it. It would be nice if V would expand the "file", doing what the linux shell is doing otherwise.

**Comment by JalonSolov** [comment]
Have you tried with PowerShell?
